counterparty-core/counterpartycore/server.py

```diff
@@ -132,6 +132,39 @@
         {"default": "localhost", "help": "the hostname or IP of the indexd server"},
     ],
     [("--indexd-port",), {"type": int, "help": "the indexd server port to connect to"}],
+    [
+        ("--api-host",),
+        {
+            "default": "localhost",
+            "help": "the IP of the interface to bind to for providing API access (0.0.0.0 for all interfaces)",
+        },
+    ],
+    [
+        ("--api-port",),
+        {"type": int, "help": f"port on which to provide the {config.APP_NAME} API"},
+    ],
+    [
+        ("--api-user",),
+        {
+            "default": "rpc",
+            "help": f"required username to use the {config.APP_NAME} API (via HTTP basic auth)",
+        },
+    ],
+    [
+        ("--api-password",),
+        {
+            "default": "rpc",
+            "help": f"required password (for --api-user) to use the {config.APP_NAME}  API (via HTTP basic auth)",
+        },
+    ],
+    [
+        ("--api-no-allow-cors",),
+        {"action": "store_true", "default": False, "help": "allow ajax cross domain request"},
+    ],
+    [
```

Comment:
interesting! why should this be configurable?

Reply:
and I think maybe it should be 503...

Reply:
> interesting! why should this be configurable?

Because I think 202 is good, but I was sure and certain that someone would think otherwise LOL. I will set 503 as default.

Reply:
done